Rewrite

revolgy-rud opened the pull request.

revolgy-rud committed:
rewrite, outline

revolgy-rud committed:
Describe git and terraform

revolgy-rud committed:
Describe strategies

revolgy-rud committed:
TBD Definitions and Abstract

revolgy-rud committed:
TBD applied

revolgy-rud committed:
TBD: wrap longer comments

revolgy-rud committed:
TBD: consistent periods in comments

revolgy-rud committed:
Code Review

revolgy-rud committed:
Code Review applied

revolgy-rud committed:
Remove about

revolgy-rud committed:
Terraform (sans CI/CD)

revolgy-rud committed:
CI/CD

revolgy-rud commented on README.md:
```diff
@@ -1,267 +1,650 @@
+
+At Revolgy, we have elected to use the following procedures, processes, or strategies.
+
+- [Trunk Based Development]
```
Hmm, I think we need @MichalReznickyRevolgy for that, as I didn't choose it.

MichalReznickyRevolgy replied:
here's the reasoning, but might be obsolete now: https://www.notion.so/revolgy/Branching-strategy-guide-f3c79363bfe643289b0874d61b3c0b6a#66c55d5084e048d6a5a27d1a244b5ca1

revolgy-rud replied:
Honestly I'll just add my own, what's written there isn't even motivation, much less reasoning.

revolgy-rud committed:
Remove empty lines in examples

Resolves:
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706727373
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706727563

revolgy-rud committed:
Fix broken links

Resolves:
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706738112
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706736427
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706746220

revolgy-rud commented on README.md:
```diff
@@ -1,267 +1,650 @@
-
-**New dependencies:**
-- dependency : dependency details
+# Revolgy Development Manifesto
```
I would rather not, we would have to think of a tool to generate it. It should be a part of the viewer, and GitHub actually has it.
![image](https://github.com/user-attachments/assets/23368ba7-9964-4e8e-add1-04a5daf613e3)

revolgy-rud commented on README.md:
```diff
@@ -1,267 +1,650 @@
+
+**Every** Terraform code base, regardless of whether it is a project or a module, should have a README.
+
+Use the [terraform-docs] command to generate the strictly technical parts of it. This process can (and should) be automated. [Link](https://github.com/terraform-docs/terraform-docs/blob/master/docs/USER_GUIDE.md#integrating-with-your-terraform-repository)
```
Please elaborate, what do you mean?

revolgy-rud committed:
Fix code-link

Resolves:
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706741807

revolgy-rud committed:
Add reasonings for tech choices

Resolves:
https://github.com/Revolgy-Business-Solutions/development-manifesto/pull/6#discussion_r1706731179
